etna/models/base.py

```diff
@@ -146,3 +148,82 @@ def _build_models(self):
         self._models = {}
         for segment in self._segments:
             self._models[segment] = deepcopy(self._base_model)
+
+
+class FitAbstractModel(ABC):
+    """Interface for model with fit method."""
+
+    @abstractmethod
+    def fit(self, ts: TSDataset) -> "FitAbstractModel":
+        """Fit model.
+
+        Parameters
+        ----------
+        ts:
+            Dataset with features
+
+        Returns
+        -------
+        result:
+            Model after fit
+        """
+        pass
+
+    @abstractmethod
+    def get_model(self) -> Dict[str, Any]:
```

Comment:
I am not sure that we actually need this method here, may be it is better to move it to the base classes for per-segment/multi-segment models